frontend/src/constants.ts

```diff
@@ -1,19 +1,47 @@
 import { CardValue } from './types/WebSocket';
 
+export const TNG_URL = 'https://www.tngtech.com/en';
+
+// Special card values
+export const VOTE_COFFEE = 'coffee';
```

Comment:
what about string enum here?

Reply:
Ja vielleicht. Wobei ich technisch kein großer Fan davon bin. Aktuell werden all diese Konstanten vom minified geinlined. Bei einem Enum hingegen sieht der generierte Code etwa so aus:

```js
// export const enum COLUMN {
//  name = 'Name',
//  vote = 'Vote',
//  voted = 'Voted',
//}
var COLUMN;
(function(COLUMN2) {
  COLUMN2["name"] = "Name";
  COLUMN2["vote"] = "Vote";
  COLUMN2["voted"] = "Voted";
})(COLUMN || (COLUMN = {}));
```

nach dem Minifier wird daraus

```js
var C, S;
((S = C || (C = {})).name = 'Name'), (S.vote = 'Vote'), (S.voted = 'Voted');
```

Geinlined wird nun nicht mehr, im Gegenteil, statt dessen hat man da jetzt überall einen Property access. Und die Namen der Properties können auch nicht minifixiert werden. Also ja, aber ich würde das lieber durch gute Namen lösen.

Reply:
Thx for the explaination